buildspec.yml

```diff
@@ -31,10 +31,34 @@ phases:
            echo "skipping notebook test"
          fi
 
+      # wait its turn to run integration tests
+      - ACCOUNT=$(aws sts get-caller-identity --output text | awk '{print $1}')
+      - S3_BUCKET_DIR=s3://sagemaker-us-west-2-${ACCOUNT}/ci-lock
+
+      - |
+        while true; do
```

Comment:
To be safe, can we put a timeout here? Some multiple of days (2-3)?

Reply:
Code build already have a timeout, the maximum run size is 8 hours.